File: python/hello_world2.py
from other_file import get_message
from car import Car

print(get_message())

car = Car()

print(car.honk())
print(car.honk())
print(car.honk())

print(car.honk_multiple_times(8))

def working_with_lists():
    arr = [
        "foo",
        "bar"
    ]

    for item in arr:
        print(item)

    for i in range(len(arr)):
        print(str(i) + ": " + arr[i])

    foo = 'foo'

    dictionary = {
        foo: "bar",
        "key": "value"
    }

    print(dictionary['foo'])
    print(dictionary['key'])

working_with_lists()

File: ruby/hello_world2.rb
require "ostruct"
require_relative "other_file"
require_relative "car"

puts getMessage()

car = Car.new

puts car.honk
puts car.honk
puts car.honk

puts car.honkMultipleTimes 8

def workingWithLists
    arr = Array.new
    # also can be used:
    # arr = [ 'foo', 'foo2' ]

    arr.push('foo')
    arr << 'foo2'

    arr.each { |item| puts item }

    for i in 0..(arr.length - 1)
        puts "#{i}: #{arr[i]}"
    end

    obj = OpenStruct.new(:foo => 'bar', :key => 'value')

    obj.baz = 'quux'

    puts obj.foo
    puts obj.key
    puts obj.baz
end

workingWithLists